tests/multi_output_models.py

```diff
@@ -0,0 +1,131 @@
+import numpy as np
+
+from sklearn.base import clone
+from sklearn.pipeline import make_pipeline
+from sklearn.preprocessing import OneHotEncoder, OrdinalEncoder
+from tensorflow.python.keras.losses import is_categorical_crossentropy
+
+from scikeras.utils.transformers import Ensure2DTransformer
+from scikeras.wrappers import BaseWrapper, KerasClassifier
+
+
+class MultiOuputClassifier(KerasClassifier):
```

Comment:
These are copies of the old implementations to handle "multilabel-indicator" and "multiclass-multioutput". Some of our tests need them, and since they are now de-implemented in plain KerasClassifier I implemented them here separately. This + model building in `test_input_ouputs.py` would end up in example notebooks in some form.

Reply:
> This + model building in test_input_ouputs.py would end up in example notebooks in some form.

:+1: I'd like to see the example notebooks in the documentation too.

Reply:
Yup, the plan is to have something like this: https://skorch.readthedocs.io/en/latest/user/tutorials.html